src/style-dictionary-dist2/TerrawareTheme2.ts

```diff
@@ -445,19 +445,19 @@ const TerrawareTheme2: ThemeDictionary = {
     "TwClrBaseWhite": "#FFFFFF",
     "TwClrBg": "#FFFFFF",
     "TwClrBgActive": "#F1F0EC",
-    "TwClrBgGhost": "#FFFFFF",
-    "TwClrBgGhostActive": "#B2AB93",
-    "TwClrBgGhostHover": "#B2AB93",
+    "TwClrBgGhost": "rgba(#FFFFFF, 0)",
+    "TwClrBgGhostActive": "rgba(#B2AB93, 0.4)",
+    "TwClrBgGhostHover": "rgba(#B2AB93, 0.2)",
```

Comment:
@karthikbtf Wondering your thoughts on what we might do here. These theme values will not be usable from TS/JS since it doesn't accept the `rgba(hexvalue, opacity)` format.

Reply:
I think I'm missing the problem.
Will this work?
```
<Typography color={theme.palette.TwClrBgGhostHover}>test</Typography>
```
(pardon the hover for color)..

Reply:
It doesn't, unfortunately. For example, I changed a button icon fill color like so:
```typescript
  svgIconFill: {
    '& path': {
      fill: theme.palette.TwClrBgGhostHover,
    },
  },
```
and in the browser I see
<img width="248" alt="image" src="https://user-images.githubusercontent.com/114949086/203633222-f6241b13-52b5-4b63-bf62-d541e4d55273.png">

Reply:
Rather than saying it "doesn't accept" the format, I should say that it does, but passes it verbatim without transforming it into `rgba(r, g, b, a)` which the browser doesn't seem to like.

Reply:
hm, alright, i can see how that might work. i'll play with that some

Reply:
got it to work! 🎉

Reply:
woot!!! will take a look